docs/operator-guide/server-setup.md

```diff
@@ -420,3 +431,71 @@ Another common failure mode is to have the joysetup.sh script fail due to a
 pre-existing zpool on the disks. When this happens often the server-setup
 plows forward and the agentsetup fails. If you see a failure, it's always a
 good idea to check the status of the zpool early in the debugging process.
+
+## iPXE booting Compute Nodes
+
+In some situations, such as a Bare-Metal-as-a-Service (BMaaS) provider, they
+may have instances that by default only network boot on one network
```

Comment:
This sounds a bit awkward.. maybe 'they may have instances where by default a system can only network boot on one network interface. The network interface may not be the interface a Triton installation has configured.....'

Reply:
Will fix.